tests/data/entries.py

```diff
@@ -21,327 +21,359 @@ def get_time(delta_mins: int) -> datetime:
     Returns:
         {datetime}: Current time minus minutes
     """
-    return (datetime.utcnow() - timedelta(minutes=delta_mins)).replace(microsecond=0, second=0)
+    return (datetime.utcnow() - timedelta(minutes=delta_mins)).replace(microsecond=0)
 
 
-good_input_entry_with_location_input: Dict[str, Any] = {
-    "labwares": [LabwaresTableEntry(id=1, barcode="labware1", location_id=1, coordinate_id=None)],
-    "audits": [
-        AuditsTableEntry(
-            id=1,
-            auditable_id=1,
-            auditable_type="Labware",
-            user_id=1,
-            updated_at=get_time(delta_mins=3),
-        )
-    ],
-    "users": [UsersTableEntry(id=1, login="user1")],
-    "locations": [LocationsTableEntry(id=1, barcode="unordered_location1", parentage="PARENT / unordered_location1")],
-}
+class TestEntries:
```

Comment:
You can't prefix classes with `Test` in the `tests/` directory because PyTest thinks your unit tests are inside the class.